db/schema.rb

```diff
@@ -21,6 +21,7 @@
     t.string "phone_number"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "job_title"
```

Comment:
Great catch with noticing this distinction on the wireframe. Let's make sure we also update the current csv file to include a column for job title as well.

Reply:
I will add the CSV with the next PR. Thanks for taking care of this J!